posts/2020-07-30-json-logging-open-liberty-20008.adoc

```diff
@@ -0,0 +1,282 @@
+---
+layout: post
+title: "Access HTTP log fields in JSON logs and write JSON directly to system.out without wrapping on Open Liberty 20.0.0.8"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: Access HTTP log fields in JSON logs and write JSON directly to system.out without wrapping on Open Liberty 20.0.0.8 - OpenLiberty.io
+seo-description: With Open Liberty 20.0.0.8 users can now access HTTP log fields in JSON logs and also write JSON directly to system.out without wrapping in the `liberty_message` event.
```

Comment:
Can you update this wording like you did in the intro (ie "you" rather than "users" etc)?

Reply:
Done